python/sdk/merlin/batch/sink.py

```diff
@@ -146,4 +147,110 @@ def to_client_config(self) -> client.PredictionJobConfigBigquerySink:
             result_column=self._result_column,
             save_mode=client.SaveMode(self._save_mode.value),
             options=opts
+        )
+
+class MaxComputeSink(Sink):
+    """
+        Sink contract for MaxCompute to create prediction job
+    """
+
+    def __init__(self, endpoint: str,
+                 table: str,
```

Comment:
ultra nit: can we have a line break before `endpoint`? 😅

Reply:
Fixed